packages/email-components/src/components/Rating/stories/RatingProps.mjml

```diff
@@ -0,0 +1,21 @@
+<mj-table padding="0" font-size="13px" css-class="readme-table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Default</th>
+      <th>Required</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>align</code></td>
+      <td><code>left | center | right</code></td>
+      <td><code>left</code></td>
+      <td>no</td>
+      <td>Rating alignment</td>
```

Comment:
If you're about to change the README alignment description, please update it here as well.